packages/ava/src/ntv/index.ts

```diff
@@ -0,0 +1,47 @@
+import type {
+  NarrativeTextSpec,
+  HeadlineSpec,
+  SectionSpec,
+  StandardSectionSpec,
+  ParagraphSpec,
+  HeadingParagraphSpec,
+  TextParagraphSpec,
+  BulletsParagraphSpec,
+  BulletItemSpec,
+  CustomBlockElement,
+  PhraseSpec,
+  TextPhraseSpec,
+  EntityPhraseSpec,
+  CustomPhraseSpec,
+  ValueAssessment,
+  EntityType,
+  EntityMetaData,
+} from './schema';
+import type { Structure, StructureTemp, Variable } from './generate';
+
+export { ENTITY_TYPES } from './schema';
+export * from './utils';
+export * from './generate';
+
+export namespace NtvTypes {
```

Comment:
In order not to repeat the name, I had to change the original types name, which may have a lot of work to upgrade in program 🫣 @pddpd @chenluli

Reply:
Fixed. Just add NtvTypes scope like:

```
// before
import { NarrativeTextSpec } from '@antv/narrative-text-schema';

// after
import { NtvTypes } from '@antv/ava';

const spec: NtvTypes.NarrativeTextSpec = {}
```